src/components/LoadingMessage.js

```diff
@@ -0,0 +1,39 @@
+export default class LoadingMessage{
```

Comment:
Please, delete class `LoadingMessage` . You don't need to hardcode searching modals again and again inside it. It's not a good practice in `OOP`